blogs/serializers.py

```
@@ -0,0 +1,25 @@
+
+
+class BlogSerializer(serializers.ModelSerializer):
+    """Serializer for the Blog model."""
+
+    created_by_name = serializers.SerializerMethodField()
+
+    class Meta:
+        """Meta class for the BlogSerializer."""
+
+        model = Blog
+        exclude = ["created_at", "updated_at"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)